static/lib/settings.js

```diff
@@ -77,7 +77,7 @@ define('forum/account/2factor', ['csrf'], function(csrf) {
 	};
 
 	Settings.generateBackupCodes = function() {
-		$.ajax('/forum/login/2fa/backup', {
+		$.ajax('/login/2fa/backup', {
```

Comment:
Replace `/forum` with `RELATIVE_PATH` please :smile: 

Otherwise this breaks on subfolder installs, but thanks for spotting this error!